src/components/Footer.tsx

```diff
@@ -34,6 +34,16 @@ export const Footer = () => {
             >
               Privacy Policy
             </TextLink>
+
+            <TextLink
+              variant={TextLinkVariant.secondary}
+              href="https://github.com/stellar/stellar-demo-wallet"
+              rel="noreferrer"
+              target="_blank"
+              title="Check out our GitHub repo for more information and to log issues"
```

Comment:
I think this should be `the`, rather than `our`. Feels too personal.
```suggestion
              title="Check out the GitHub repo for more information and to log issues"
```

What 'more information' are we hoping to give to folks? Who is the audience of this call to action?

I defer to whoever is deciding about copy. 😄

Reply:
@leighmcculloch the main audience to date have been anchor developers testing implementation against a mock client (this). That's still the aim as well as extending this to a learning tool. We don't want the full banner at the top of the page but we do want users to know they can log bugs in this repo. A README is here as well although we need to float instructions on the main page as well.